db/migrate/20210603171313_create_dttp_schools.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateDttpSchools < ActiveRecord::Migration[6.1]
+  def change
+    create_table :dttp_schools do |t|
+      t.string :name
+      t.string :dttp_id, index: { unique: true }
+      t.string :urn
+
+      t.timestamps default: -> { "CURRENT_TIMESTAMP" }
```

Comment:
I don't think adding the `default` option is necessary here as `created_at` and `updated_at` are automatically populated by Active Record.

Reply:
This was added intentionally since we are using [`upsert_all`](https://apidock.com/rails/v6.0.0/ActiveRecord/Persistence/ClassMethods/upsert_all) for a mass update, it doesn't initialize AR stuff, and raises a `PG::NotNullViolation` exception.